app/gui2/src/components/ExecutionModeSelector.vue

```diff
@@ -112,4 +115,17 @@ span {
     background: var(--color-dim);
   }
 }
+
+.dropdown-enter-active,
+.dropdown-leave-active {
+  transition: all 0.1s;
+}
+
+.dropdown-enter-from,
+.dropdown-leave-to {
+  max-height: 0;
+  padding: 0;
```

Comment:
```suggestion
  padding-top: 0;
  padding-bottom: 0;
```

not sure if this can be replaced with the shorthand, as we don't want to override the existing horizontal padding.